src/index.js

```diff
@@ -1,3 +1,5 @@
 import './common.css'
 import './index.css'
 import './checklist.css'
+import '../public/listiq.svg'
```

Comment:
I don't really like this, but having `svg` files in `src` folder doesn't appeal to me.